application.py

```diff
@@ -17,112 +18,175 @@
 # AWS Elastic Beanstalk looks for application by default,
 # if this variable (application) isn't set you will get a WSGI error.
 application = app.server
-
```

Comment:
I moved this out of the application file because it's more about layout / lookups, and we should do that in other apps in the future.